src/main/resources/db/changelog/changes/06-create-category-table.yaml

```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
   - changeSet:
       id: create-category-table
-      author: anton
```

Comment:
technically you should not update the existing changesets. But since you haven't deployed your app, you can do it locally